src/test-utils/dom/internal/dropdown-host.ts

```diff
@@ -161,7 +161,7 @@ export class DropdownContentWrapper extends ComponentWrapper {
   }
 
   findHighlightedAriaLiveRegion(): ElementWrapper | null {
-    return this.find('[aria-live]');
+    return this.findHighlightedOption()?.findByClassName(selectableStyles['screenreader-content']) ?? null;
```

Comment:
This method is repeated in two places: `src/test-utils/dom/autosuggest/index.ts` and `src/test-utils/dom/internal/dropdown-host.ts`, but so are a bunch of other methods, so I would like to refactor separately after this PR.